### DISCUSSION TITLE
op.algorithms.InvasionPercolation can not output to paraview

### DISCUSSION THREAD

**Discussion by Xknightfyy** [PMEAL/OpenPNM#2717] [Q&A]
Dear Professor Jeff Gostick,

I hope this email finds you well. I appreciate your previous responses and I am reaching out to you regarding a new issue I have encountered. After performing the IP algorithm, I am unable to export the file to Paraview. I have not come across this error before and I am unsure how to address it. Could you please assist me with this matter? Thank you in advance for your help.

Best regards, [xu]
![671aa790fc16ac4d0a681bc7cb1ddff](https://user-images.githubusercontent.com/120148734/228822779-34d5e8e2-05c4-4d91-8329-7e3a98f1a0f5.png)
![4aecf76d88d556068f8cb894d066923](https://user-images.githubusercontent.com/120148734/228822797-e6a8c18a-caaa-42d0-8da0-b68de6dded21.png)

**Comment by jgostick** [comment]
We fixed this error in a recent update, can you do ``pip install --upgrade openpnm`` and see if it goes away?

**Reply by Xknightfyy** [reply]
Yes, it's gone! Thank you very much!
